imapexecute/__main__.py

```diff
@@ -19,7 +20,25 @@ def start_connection(name, config):
     else:
         connections[name] = imaplib.IMAP4(config['host'], port=int(config['port']))
 
-    connections[name].login(config['username'], config['password'])
+    password = None
+    if config.get('password'):
+        password = config['password']
+    else:
+        if config.get('password_cmd'):
+            p = subprocess.Popen(config['password_cmd'].split(),
```

Comment:
This is not correct for a variety of commands, might be better to not split it and use shell=True

Reply:
Which commands would this not work for?

I considered shell=True, but I try to avoid that as much as possible...

Reply:
Any command with quotes in it will be split inside the quotes and then assembed wrong again by popen, in this case shell=True is correct